lib/src/typescript/node.ts

```diff
@@ -208,9 +211,9 @@ const metaLength = 1 + 64 + 16 + 32
  */
 export const decrypt = (privateKey: Buffer, encrypted: Buffer): Promise<Buffer> => new Promise((resolve, reject) => {
     if (encrypted.length < metaLength)
-        reject(new Error(`Invalid Ciphertext. Data is too small. It should ba at least ${metaLength}`))
-    else if (encrypted[0] !== 4)
-        reject(new Error('Not valid ciphertext. A valid ciphertext would begin with 4'))
+      reject(new Error(`Invalid Ciphertext. Data is too small. It should ba at least ${metaLength}`))
+    else if (encrypted[0] !== 2 && encrypted[0] !== 3 && encrypted[0] !== 4)
+      reject(new Error(`Not a valid ciphertext. It should begin with 2, 3 or 4 but actualy begin with ${encrypted[0]}`))
```

Comment:
`actually` instead of `actualy`

Reply:
To me, the first byte should only equal 4 (not even 2 or 3).

Reply:
To be frank, I did not really think this through, I just follow ecies parity on this. It does make sense as we test this in derive. I will change it and we will see.